subprojects/core/src/integTest/groovy/org/gradle/api/internal/tasks/TaskCacheabilityReasonIntegrationTest.groovy

```diff
@@ -339,7 +339,7 @@ class TaskCacheabilityReasonIntegrationTest extends AbstractIntegrationSpec impl
 
         then:
         withBuildCache().succeeds("producer", "consumer")
-        assertCachingDisabledFor VALIDATION_FAILURE, "Validation failed", ":consumer"
+        assertCachingDisabledFor VALIDATION_FAILURE, "Caching has been disabled to ensure correctness due to validation failures. Please consult deprecation warnings for more details.", ":consumer"
```

Comment:
```suggestion
        assertCachingDisabledFor VALIDATION_FAILURE, "Caching has been disabled due to validation failures to ensure correctness. Please consult deprecation warnings for more details.", ":consumer"
```

It sounds much better to me if the `due to` comes first.

Reply:
I agree, changed it so.